frontend/src/routes/(authenticated)/project/[project_code]/viewer/lfClassicLexboxApi.ts

```diff
@@ -10,7 +10,25 @@
   type PartOfSpeech,
   type SemanticDomain
 } from 'viewer/lexbox-api';
+import { semanticDomains_en } from './semantic-domains.en.generated-data';
+
+function prepareEntriesForUi(entries: IEntry[]): void {
+  entries.forEach(entry => {
+    entry.senses.forEach(sense => {
+      sense.semanticDomains.forEach(sd => {
+        sd.id = sd.code;
+      });
+      // @ts-expect-error
+      sense.partOfSpeechId = sense.partOfSpeech;
+    });
+  });
+}
 
+function preparePartsOfSpeedForUi(partsOfSpeech: PartOfSpeech[]): void {
+  partsOfSpeech.forEach(pos => {
+    pos.id = pos.name['__key'];
```

Comment:
I don't see anywhere that this value is defined. Am I missing something?

Reply:
Good catch. I think I lost that change as I worked on the filtering.